bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/servlets/AdaptiveImageServlet.java

```diff
@@ -736,8 +736,8 @@ private Integer getAllowedJpegQuality(@NotNull Resource imageResource) {
     private long getRequestLastModifiedSuffix(@Nullable String suffix) {
         long requestLastModified = 0;
         if (StringUtils.isNotEmpty(suffix) && suffix.contains(".")) {
-            // check if the 13 digits UTC milliseconds timestamp is present in the suffix
-            Pattern p = Pattern.compile("\\(|\\)|\\d{13}");
+            // check if the 13 digits UTC milliseconds timestamp, proceeded by a forward slash is present in the suffix
+            Pattern p = Pattern.compile("\\(|\\)|\\/\\d{13}");
```

Comment:
typo? Shouldn't it be proceeded -> preceded?

Reply:
revolved in 218fac0354c2d75031880b3ac0888455f621c9e4